site/content/docs/user/ingress.md

```diff
@@ -52,15 +56,16 @@ nodes:
 EOF
 {{< /codeFromInline >}}
 
-If you want to run with multiple nodes you must ensure that your ingress-controller is deployed on the same node where you have configured the PortMapping, in this example you can use a [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) to specify the control-plane node name.
-
-{{< codeFromInline lang="yaml" >}}
-nodeSelector:
+For multi-node clusters with `extraPortMapping`:
+- The built-in Ingress (via cloud-provider-kind, since v0.9.0) does not rely on `extraPortMapping` and works independently.
+- If you explicitly use a third-party Ingress controller (not recommended for most users), you may need to configure `nodeSelector` to ensure the controller runs on the node with port mappings:
+  {{< codeFromInline lang="yaml" >}}
+  nodeSelector:
   kubernetes.io/hostname: "kind-control-plane"
-{{< /codeFromInline >}}
+  {{< /codeFromInline >}}
 
 ### Ingress NGINX
```

Comment:
can we please remove this section now that the retirement is official ?

https://kubernetes.io/blog/2025/11/11/ingress-nginx-retirement/